calculator.js

```diff
@@ -0,0 +1,123 @@
+// Get the user inputs
+
+var doMath = prompt("Type anything to do math!");
```

Comment:
This is a little confusing for the user and isn't necessary. I wouldn't include this.